iam_builder/iam_schema.py

```diff
@@ -0,0 +1,61 @@
+import jsonschema
+
+IAM_SCHEMA = {
```

Comment:
Could you make this a json file and load it in please? [the same way data linter does](https://github.com/moj-analytical-services/data_linter/blob/main/data_linter/constants.py).
You should just be able to use `json.dump` to dump this dictionary to a json file and save yourself the work

Reply:
Done - cbff1e4